sql/core/src/test/resources/sql-tests/analyzer-results/pipe-operators.sql.out

```diff
@@ -2769,6 +2741,239 @@ org.apache.spark.sql.catalyst.ExtendedAnalysisException
+      +- Project [cate#x, val#x]
+         +- SubqueryAlias windowtestdata
+            +- View (`windowTestData`, [val#x, val_long#xL, val_double#x, val_date#x, val_timestamp#x, cate#x])
+               +- Project [cast(val#x as int) AS val#x, cast(val_long#xL as bigint) AS val_long#xL, cast(val_double#x as double) AS val_double#x, cast(val_date#x as date) AS val_date#x, cast(val_timestamp#x as timestamp) AS val_timestamp#x, cast(cate#x as string) AS cate#x]
+                  +- Project [val#x, val_long#xL, val_double#x, val_date#x, val_timestamp#x, cate#x]
+                     +- SubqueryAlias testData
+                        +- LocalRelation [val#x, val_long#xL, val_double#x, val_date#x, val_timestamp#x, cate#x]
+
+
+-- !query
+table windowTestData
+|> select cate, val, first_value(cate) over w as first_val
+|> select cate, val, sum(val) over w as sum_val
+   window w as (order by val)
+-- !query analysis
+Project [cate#x, val#x, sum_val#xL]
++- Project [cate#x, val#x, _we0#xL, pipeselect(_we0#xL) AS sum_val#xL]
+   +- Window [sum(val#x) windowspecdefinition(val#x ASC NULLS FIRST, specifiedwindowframe(RangeFrame, unboundedpreceding$(), currentrow$())) AS _we0#xL], [val#x ASC NULLS FIRST]
+      +- Project [cate#x, val#x]
+         +- Project [cate#x, val#x, first_val#x]
+            +- Project [cate#x, val#x, _we0#x, pipeselect(_we0#x) AS first_val#x]
+               +- Window [first_value(cate#x, false) windowspecdefinition(val#x ASC NULLS FIRST, specifiedwindowframe(RangeFrame, unboundedpreceding$(), currentrow$())) AS _we0#x], [val#x ASC NULLS FIRST]
+                  +- Project [cate#x, val#x]
+                     +- SubqueryAlias windowtestdata
+                        +- View (`windowTestData`, [val#x, val_long#xL, val_double#x, val_date#x, val_timestamp#x, cate#x])
+                           +- Project [cast(val#x as int) AS val#x, cast(val_long#xL as bigint) AS val_long#xL, cast(val_double#x as double) AS val_double#x, cast(val_date#x as date) AS val_date#x, cast(val_timestamp#x as timestamp) AS val_timestamp#x, cast(cate#x as string) AS cate#x]
+                              +- Project [val#x, val_long#xL, val_double#x, val_date#x, val_timestamp#x, cate#x]
+                                 +- SubqueryAlias testData
+                                    +- LocalRelation [val#x, val_long#xL, val_double#x, val_date#x, val_timestamp#x, cate#x]
+
+
+-- !query
+table windowTestData
+|> select cate, sum(val) over w
+-- !query analysis
+org.apache.spark.sql.catalyst.ExtendedAnalysisException
+{
+  "errorClass" : "MISSING_WINDOW_SPECIFICATION",
+  "sqlState" : "42P20",
+  "messageParameters" : {
+    "docroot" : "https://spark.apache.org/docs/latest",
+    "windowName" : "w"
+  }
+}
+
+
+-- !query
+table windowTestData
+|> select cate, val, sum(val) over w1, first_value(cate) over w2
+   window w1 as (partition by cate)
+   window w2 as (order by val)
```

Comment:
I'm a bit confused, aren't `w1` and `w2` both defined?

Reply:
Similar to the classic SQL, when mulitple WINDOW clauses are presented, the SQL parser will only see the first window definition.

![image](https://github.com/user-attachments/assets/1d8f55be-0698-4035-9bb9-17e1a4237f81)

Reply:
hmm, is it a bug? @srielau

Reply:
If it is, it seems like the same bug for both regular SQL syntax and pipe operators, and the behavior would work the same for both (now, and if we were to change it later).